src/styles/_reset.scss

```
@@ -0,0 +1,4 @@
+* {
+  margin: 0;
+  box-sizing: border-box;
+}
```

наскільки пам ятаю це не відноситься до reset
```suggestion
  box-sizing: border-box;
}
```